### ISSUE TITLE
Japan OSM improvement

### ISSUE THREAD

**Issue by planemad** [mapbox/mapping#120]
Based on observations from #116. There are multiple data issues in japan which requires a comprehensive cleanup strategy with close involvement of the Japanese OSM community.

This ticket will track all the high level conversations and strategy on improving OSM data for Japan.

**Existing data issues**
- Most of Japan road network has been imported from [Yahoo Japan](https://wiki.openstreetmap.org/wiki/YahooJapanALPS_Data) in 2011
- The road data has positional errors (5-30m) when compared to Strava GPS and includes incorrect topology that does not match satellite imagery. 
![screenshot 2015-09-14 16 08 10](https://cloud.githubusercontent.com/assets/126868/9848018/16328874-5afb-11e5-8fb4-a4eb52953b9a.png)
- Roads in metropolitan areas have been realigned to the correct position, but large parts of the country are still untouched since 2011.
![screenshot 2015-09-14 16 27 36](https://cloud.githubusercontent.com/assets/126868/9848298/8ec5afa8-5afd-11e5-9c7f-39b230ea85a1.png)
- The Bing imagery coverage for Japan is comprehensive but does not match the OSM data or Strava GPS data. There is both an offset and orthorectification errors that varies throughout Japan.
![](https://cloud.githubusercontent.com/assets/10141319/9680104/02624708-5312-11e5-85f5-b0f944405309.png)
- There is high resolution orthorectified imagery for Japan from GSI which perfectly matches Strava GPS data and is the best imagery source available. The coverage of this imagery is not complete.
![](https://cloud.githubusercontent.com/assets/126868/9682282/dccb1d38-5322-11e5-97c4-ab57ba45cf5e.png)

**Data improvement strategy**

1. Select project area
   1. Priority: Urban areas > Highway corridors > Rural areas
   2. Check the accuracy of orthorectified imagery and Bing imagery by comparing with GPS data. Publish offsets if required.
   3. Evaluate extent of data offset and data freshness for the area
   4. Publish findings and communicate with local mappers to initiate a mapping task
2. Road realignment
  1. Create a road improvement task using the [tasking manager](http://tasks.openstreetmap.in)
  2. Verify imagery alignment using GPS data
  3. For gridded networks, attempt to offset data to match imagery. If not successful delete imported unnamed roads and retrace area
  4. Delete imported roads with no match to imagery
  5. Realign all other roads and mark as finished
3. Road tracing
  1. Go through realigned tasks and invalidate to commence road tracing
  2. Trace missing roads and tracks from imagery and mark as finished
4. Validation

**Next steps**
- [x] Get feedback from Japan OSM community on above process
- [x] @planemad to set up trial map improvement task: http://tasks.openstreetmap.in/project/38
- [x] @planemad and @Rub21 to run task with team and closely monitor issues

cc @mapbox/team-data

**Comment by planemad** [comment]
**Realignment trial report 09/16**
- Today @pratikyadav @ruthmaben @nikhilprabhakar and @ramyaragupathy realigned 15,194 nodes in the [Mutsu area](http://tasks.openstreetmap.in/project/38) 
- Initial pass was to realign all roads in the task area, but due to inconsistencies in the [reference sources](https://github.com/mapbox/mapping/issues/116#issuecomment-140357569) and lack of GSI satellite imagery coverage, we could not ascertain that our work was accurate.
- Our revised strategy was to realign only highways (motorway-secondary) based on Strava GPS which was more accurate, but this created inconsistencies wrt to the existing data
- In the interest of keeping the improvement process simple for the community to contribute to, the conclusion was to sacrifice small positional accuracies to maintain large scale data consistency.

**Revised data improvement strategy**
1. Focus on high priority areas based on @Rub21 [tweet map](http://rub21.github.io/top-tiles/app/)
![image](https://cloud.githubusercontent.com/assets/1152236/9856571/24ad4c7e-5adb-11e5-8af0-0ad3ea7f5bb2.png)
2. Use the primary tracing source as `GSI Japan Satellite (ort)` where available and fallback on `Bing` in other areas. Most high priority areas [have GSI Satellite coverage](https://api.mapbox.com/v4/pratikyadav.79cb6a87/page.html?access_token=pk.eyJ1IjoicHJhdGlreWFkYXYiLCJhIjoiMTA2YWUxNjRkNmFmZGQ4YzAxZWFiNDk0NDM1YjE1YjAifQ.4P6N5dNmA_WQXd3BsJvu5w#6/37.222/139.724).
<img width="762" alt="screenshot 2015-09-16 19 40 57" src="https://cloud.githubusercontent.com/assets/126868/9907044/eaec3898-5caa-11e5-8e16-600736b5aeb7.png">
3. Realign road network only in urban areas to `GSI Japan Satellite (ort)`, ignore offsets wrt `Bing`
4. Trace missing roads based on tracing source

From tomorrow team-blr will try the above approach and report findings. A log of observations is maintained [here](https://textb.org/t/mbx-data-japan/)

cc @mapbox/team-data

**Comment by lxbarth** [comment]
Wow, great progress.

> In the interest of keeping the improvement process simple for the community to contribute to, the conclusion was to sacrifice small positional accuracies to maintain large scale data consistency.

@planemad - what does this concretely mean? Can you give an example for that?

**Comment by planemad** [comment]
@lxbarth to be as accurate as possible we were using Strava and OSM GPS to offset the Bing/GSI imagery for the best match, this is never perfect and results in an offset in another area. These continuous localized corrections which cannot be replicated exactly be someone else and adds to the data inconsistency since no source perfectly will match the data everywhere.

To make it simple, we're just going to use GSI Satellite or Bing as is without offset correction. We will skip realigning data with offsets under 10m from satellite imagery such as these and focus on adding missing roads.
<img width="512" alt="screenshot 2015-09-16 20 58 10" src="https://cloud.githubusercontent.com/assets/126868/9909431/cfce098c-5cb5-11e5-88a8-e3f838527c41.png">

**Comment by mapconcierge** [comment]
@planemad 

I understand all steps. 
Yes, GSI ortho image layer (ort) has good accuracy.

We have started special project team on Facebook Group with Asian addicted mappers, OSM teachers in Japan. My students will join.
https://www.facebook.com/groups/697237443754399/

**Comment by mapconcierge** [comment]
@planemad 

Mutsu city area doesn't have good GSI ort images and Strava GPS.
We should choice 

std 標準地図 MaxZoomLv18
http://cyberjapandata.gsi.go.jp/xyz/std/{z}/{x}/{y}.png
source=GSImaps/std

![2015-09-17 13 43 07](https://cloud.githubusercontent.com/assets/416977/9925104/1736f57c-5d42-11e5-84f5-81bd4a7c1f5d.png)

**Comment by pratikyadav** [comment]
Today  @planemad @pratikyadav @ruthmaben @nikhilprabhakar and @ramyaragupathy realigned 6437 nodes in the [ Aomori region](http://tasks.openstreetmap.in/project/40#task/42) for major roads.

The alignment was based on Japan GSI Satellite imagery.

![screen shot 2015-09-17 at 6 30 16 pm](https://cloud.githubusercontent.com/assets/12744420/9933725/6ac07966-5d6a-11e5-9a04-70630949b799.png)

**Comment by nikhilprabhakar** [comment]
Hi @mapconcierge ,
What kind of highway will the below road be?
Confused on whether this will be a service road, residential road or a living-street road.
https://www.openstreetmap.org/#map=17/40.785891873025996/140.7407283726662
![japan](https://cloud.githubusercontent.com/assets/11845908/9988885/f4e485e2-6071-11e5-85ab-855fbb78e430.gif)

**Comment by pratikyadav** [comment]
#### Realignment trial report 09/21

Today @planemad @pratikyadav @ruthmaben @nikhilprabhakar and @ramyaragupathy realigned 3201 nodes in the [ Aomori region](http://tasks.openstreetmap.in/project/40#task/42).

The focus under pass 1A and 2 was - 
- to create dual carriageways on major highways, 
- realign and merge tertiary
- Adding missing tertiary+ highways
- Adding missing bridges/tunnels on tertiary+ highways;

Today under pass 3, the team will realign, merge and add missing unclassified, residential roads, living streets.

![screen shot 2015-09-18 at 6 15 54 pm](https://cloud.githubusercontent.com/assets/12744420/9991070/63504ebe-6084-11e5-8f0f-2a59c28b456c.png)

**Comment by planemad** [comment]
**Realignment Summary**
- Team of @pratikyadav @ruthmaben @nikhilprabhakar and @ramyaragupathy conducted a multi phase data correction exercise in Aomori
- We synced daily to break the data correction in multiple steps to prioritize effort to the most critical data first to least important and improve the process from our observations
- General observations:
  - Time taken to realign an object is proportional to the number of ways and nodes
  - By merging consecutive ways and removing unnecessary nodes in straight sections, the effort can be reduced
  - In areas with dense road network, it is faster to delete roads without extra properties (name, ref) and retrace than realign
  - Focusing a task on a limited set of features allows the use of filters to remove distracting objects
  - Time per user per task should be limited to under 60 mins, else there is a feeling of stagnation
  - Retagging roads should not be done without valid reason
  - At all points the larger objective should be to simplify the contribution of any newbie to OSM in Japan

**Realignment proposal for Japan**
Based on these finding this is the proposal for a large scale road network correction for Japan:

- Select a project area with GSI imagery coverage and create a mapping task
- Notify OSM-JP of the task
- Make multiple passes of each area and make improvements in steps:
  - Pass 1: Realign and merge major highways (motorway-secondary);  Create dual carriageways and link roads with correct oneways
  - Pass 2: Realign and merge tertiary; Adding missing tertiary+ highways and link roads; Adding missing bridges/tunnels on tertiary+ highways;
  - Pass 3: Realign, merge and add missing unclassified, residential roads, living streets
  - Pass 4: Realign, merge and add missing service roads, tracks and paths
  - Pass 5: Correct natural features polygons and waterways, add missing bridges, tunnels and viaducts
- The @mapbox/team-data will focus on completion upto pass 3 throughout Japan

**Comment by planemad** [comment]
Since we have Aomori completed till pass 2, the @mapbox/team-data is going to follow the similar procedure in the more dense Beppu-Oita region on Kyushu island from today http://tasks.openstreetmap.in/project/44

![](http://foreign.beppu-navi.jp/CMSimage/photo/20120127_707475097_FilePath0.jpg)

Getting in touch with the OSM-JP community through @mapconcierge to help in validation of previous work and provide feedback on the process.

**Comment by planemad** [comment]
While realigning highway junctions, be careful about fixing any kinks.
![untitled](https://cloud.githubusercontent.com/assets/126868/10012730/2f3d18ec-6128-11e5-8315-8019a2775b91.gif)

**Comment by nikhilprabhakar** [comment]
@mapbox/team-data  - Also merge and fix the junctions which are split like in the below.
![junctions](https://cloud.githubusercontent.com/assets/11845908/10013061/8edc08c8-612b-11e5-95d3-b5a612371f12.gif)

**Comment by samely** [comment]
We already finished http://tasks.openstreetmap.in/project/31#. Those are the number of editions per user

User | Num OSM Objects | Num Changeset | Num nodes V1 | Num nodes Vx | Num ways V1| Num ways Vx | Num relation V1 | Num relation Vx 
 ---|---|---|---|---|---|---|---|--- 
samely | 77,862 | 333 | 45,220 | 18,416 | 5,856 | 8,370 | 0 | 0
RichRico | 65,217 | 441 | 31,035 | 28,070 | 2,511 | 3,601 | 0 | 0
karitotp | 64,290 | 215 | 32,538 | 21,848 | 4,531 | 5,370 | 0 | 3
Luis36995 | 53,561 | 429 | 24,981 | 21,864 | 2,751 | 3,965 | 0 | 0
calfarome | 40,985 | 309 | 19,079 | 16,561 | 2,624 | 2,721 | 0 | 0
dannykath | 38,549 | 653 | 20,531 | 11,481 | 3,563 | 2,973 | 0 | 1
ediyes | 32,781 | 540 | 15,149 | 12,291 | 2,467 | 2,869 | 0 | 5
andygol | 10,087 | 24 | 2,007 | 7,382 | 240 | 455 | 2 | 1
abel801 | 31,879 | 289 | 9,912 | 19,519 | 638 | 1,810 | 0 | 0
Rub21 | 1,815 | 31 | 759 | 711 | 131 | 214 | 0 | 0
pratikyadav | 986 | 26 | 693 | 76 | 158 | 59 | 0 | 0
ruthmaben | 767 | 18 | 236 | 405 | 58 | 68 | 0 | 0
jinalfoflia | 311 | 7 | 240 | 5 | 52 | 14 | 0 | 0
nikhilprabhakar | 310 | 5 | 254 | 9 | 34 | 13 | 0 | 0
srividya_c | 294 | 17 | 167 | 14 | 75 | 38 | 0 | 0
ramyaragupathy | 266 | 13 | 158 | 29 | 45 | 34 | 0 | 0
Chetan_Gowda | 265 | 7 | 154 | 94 | 5 | 12 | 0 | 0
**Total** | 420,225 | 3,357 | 203,113 | 158,775 | 25,739 | 32,586 | 2 | 10

**Comment by calfarome** [comment]
Visualization the progress of the team in the peninsula [Chiba Prefecture, Japan](https://github.com/mapbox/mapping/issues/116).
![japav22l](https://cloud.githubusercontent.com/assets/10413117/10023077/cc6dba94-6115-11e5-93b0-2e275bc85459.png)

**Comment by nikhilprabhakar** [comment]
@mapbox/team-data : I have come across a lot of duplicate roads in the current task that we are doing (Ishikawa). For now we will be ignoring these duplicate roads and will be taking care of it in the later passes.
![duplicates](https://cloud.githubusercontent.com/assets/11845908/10070598/246a439a-62d1-11e5-9cbd-c6aa02c6e64a.gif)

**Comment by jinalfoflia** [comment]
@mapconcierge, there is this tunnel like construction, what should it be tagged as? 
![screen shot 2015-09-24 at 3 35 47 pm](https://cloud.githubusercontent.com/assets/4470913/10070820/9a9d5646-62d2-11e5-84ed-93591040a92a.png)

[Here](https://www.openstreetmap.org/node/1395521826/history#map=19/36.88596/136.88761) is the link to the road.

**Comment by planemad** [comment]
**Japan realignment progress map**
![screenshot 2015-09-24 16 54 08](https://cloud.githubusercontent.com/assets/126868/10072164/f898e0b2-62dc-11e5-9e80-6153caf95cda.png)
https://gist.github.com/planemad/64da69b20ea9705b5673

- Task polygons based on GSI coverage for use in tasking manager
- Stroke width indicates importance and is based on population/road density
- Color indicates task status. Red: todo, Orange: Active, Green: Done

**Comment by friedbunny** [comment]
@jinalfoflia Coincidentally, I've been through that tunnel many times. It's an enclosed "snow shed"/"snow shelter" (スノーシェッド or スノーシェルター) situated at a mountain pass with hills rising on either side. 

[Here are some pictures of this structure.](http://hidekikawa.at.webry.info/200906/article_4.html)

[It seems like](http://wiki.openstreetmap.org/w/index.php?title=Proposed_features/snowshed&oldid=450157) the OSM tagging is `covered=yes`.

**Comment by jothirnadh** [comment]
@mapbox/team-data : Whenever there is a pre-defined tunnel or a bridge such as one in the below image, we are not making any changes due to lack of evidence. 
![screen shot 2015-09-25 at 6 06 13 pm](https://cloud.githubusercontent.com/assets/8401827/10100602/85d7cd5c-63b0-11e5-8361-a87ce149f01c.png)

**Comment by planemad** [comment]
**Progress**
- Task: http://tasks.openstreetmap.in/project/45
- Area: ~8400km2 
- Total time taken: 
  - Pass 1: 3 days
  - Pass 2: active

**Comment by jinalfoflia** [comment]
@friedbunny Thank you :+1:

**Comment by jinalfoflia** [comment]
As there are multiple roads for re-aligning, I tend to miss a few of them. To better keep track of the roads that I modified, I use this [style]( https://raw.githubusercontent.com/Andygol/josm-styles/master/created_in_2015.mapcss) that highlights freshness of data. 

- Before I started editing, we can see that the roads are not highlighted.

![screen shot 2015-09-29 at 6 33 35 pm](https://cloud.githubusercontent.com/assets/4470913/10165111/0e2a0b34-66db-11e5-99d2-6f5a28bc8a13.png)

- After aligning them, I uploaded the data. After that I downloaded the same area again, this time the edits made by the me were highlighted and thus it'll be easy to spot the roads that were missed.
![screen_shot_2015-09-29_at_6_37_36_pm](https://cloud.githubusercontent.com/assets/4470913/10165190/8078d0a8-66db-11e5-8f38-d376a5bccdb5.png)

/cc @mapbox/team-data

**Comment by planemad** [comment]
@jinalfoflia nice tip and its always a good practice to upload/download data to keep it in sync.

**Comment by Rub21** [comment]
@mapbox/team-data : 
We've finished up the pass 2 in  http://tasks.openstreetmap.in/project/45, let's start in new task  http://tasks.openstreetmap.in/project/48

**Comment by mapconcierge** [comment]
> @Rub21 
OK!! I've confirm Fukushima Area.
Our students team will join today. :-)

> Rub21
Please add admin permission for Project 45 to me, I can translate in Japanese.

**Comment by Rub21** [comment]
@mapconcierge , That is great :) , could you give me  your osm users,  to add in Task Manager, hmm but I think first you have to login http://tasks.openstreetmap.in/, after it I can give you permission to access.

**Comment by lxbarth** [comment]
[![](https://www.evernote.com/shard/s210/sh/6548eb08-7039-47a2-ab7d-eaf951f0eb22/e29dc6c8b8514c58/res/6aad6a6c-0c20-4031-afd0-2ceda2481911/skitch.png?resizeSmall&width=832)](https://developmentseed.org/blog/2015/09/28/whats-trending-osm/)

**Comment by mapconcierge** [comment]
＞ @Rub
MAPconcierge 
http://www.openstreetmap.org/user/MAPconcierge


＞ @lxbarth 
That's great!! :-)

**Comment by mapconcierge** [comment]
Japanese Students are doing now!!
![12017556_1126710234024613_4210415448895765433_o](https://cloud.githubusercontent.com/assets/416977/10214888/4a1cdbf8-6857-11e5-963e-07f4c6e0c286.jpg)

**Comment by planemad** [comment]
This is amazing @mapconcierge !

Has there been any confusion from the students, maybe you can translate the instructions in Japanese for the task?

**Comment by mapconcierge** [comment]
@planemad 
Yes, of course!! :-)

**Comment by planemad** [comment]
@mapconcierge sorry, thought you were already admin, done now. Also lets have subsequent tasks on http://tasks.teachosm.org , it would be great to have the community select the area that needs to be worked on based on https://gist.github.com/planemad/64da69b20ea9705b5673

cc @Rub21

**Comment by abel801** [comment]
We don't have GSI imagery in [This](http://tasks.openstreetmap.in/project/48#task/378) I think that we have finished the task.

@planemad

**Comment by Rub21** [comment]
@mapbox/team-data , I've added other task to improve Japan, [#49 - Japan road improvement: Mito](http://tasks.openstreetmap.in/project/49) if you can not find more blocks to work in [#48 - Japan road improvement: Fukushima](http://tasks.openstreetmap.in/project/48) can you jump to this new one

**Comment by mapconcierge** [comment]
＞ @planemad 
Okay thanks, I'm doing translation in Japanese. :-)
http://tasks.openstreetmap.in/project/48

**Comment by planemad** [comment]
**Mapping report of [Ishikawa task](http://tasks.openstreetmap.in/project/45)**
- Area: 8000 km2
- Pop density: 200-300 persons/km2
- Pass 1: 3 days with 18 members (Mapbox)
- Pass 2: 1.5 days with 19 members (Mapbox)
- [Notes](https://textb.org/t/mbx-data-japan/)

**Comment by planemad** [comment]
@mapbox/team-data based on the above observations and a voice discussion, it was decided to continue mapping in separate passes for future tasks. The most important reason for this is to detect quality issues from the previous pass.

**Comment by nikhilprabhakar** [comment]
@mapbox/team-data : We are not combining the roads which have different `Maxspeed` tag value like for the below road.
![speeddif](https://cloud.githubusercontent.com/assets/11845908/10278270/8b70dbac-6b79-11e5-8ee3-06dd3c37b7dc.gif)

**Comment by pratikyadav** [comment]
![japan](https://cloud.githubusercontent.com/assets/12744420/10304414/a36fe252-6c37-11e5-9334-be281599b2da.gif)


Seeing a lot of holes/missing strips in GSI japan satellite imagery which are not actually one when we zoom in.

@mapbox/team-data

**Comment by pratikyadav** [comment]
## Mapping report 

Pass 1 and 2 done on two tasks for the parts with Japan GSI Satellite Imagery coverage.

**[48 - Japan road improvement: Fukushima](http://tasks.openstreetmap.in/project/48)**

![screen shot 2015-10-08 at 11 25 13 am](https://cloud.githubusercontent.com/assets/12744420/10358764/4beca8d4-6daf-11e5-9665-9e808aed81a6.png)


**[49 - Japan road improvement: Mito](http://tasks.openstreetmap.in/project/49)**

![screen shot 2015-10-08 at 11 25 22 am](https://cloud.githubusercontent.com/assets/12744420/10358765/50244718-6daf-11e5-8c54-96434df18a59.png)


Currently @mapbox/team-data  is working on [54 - Japan road improvement: Osaka](http://tasks.openstreetmap.in/project/54)

cc @mapconcierge

**Comment by maning** [comment]
For later consideration, it would interesting to log who is editing in the TM outside the data team.  This is just to highlight that this work is with the rest of community in Japan.

cc @mapconcierge

**Comment by planemad** [comment]
**Important**
JOSM allows combining ways with different relations without any warning. In this example, roads with the same tags but part of different bus route relations were merged with no dialog. This will break the bus route relation. Be very careful when merging roads in urban areas as they usually have bus route relations
![untitled](https://cloud.githubusercontent.com/assets/126868/10359915/5a2a9eaa-6dba-11e5-9467-9f1859fc7149.gif)

Bug report has been filed in the JOSM trac: https://josm.openstreetmap.de/ticket/11945

cc @mapbox/team-data

**Comment by Andygol** [comment]
**In order not to miss anything - use Todo Plugin**

![image](https://cloud.githubusercontent.com/assets/369696/10397293/88b3b68a-6eaf-11e5-82e4-f9e482655f73.png)
http://wiki.openstreetmap.org/wiki/JOSM/Plugins/TODO_list

1.  Select required features
2. Add them into Todo List
3. Select and mark it one by one
4. Save result
5. Clear the list

![josm-using-todo-plugin](https://cloud.githubusercontent.com/assets/369696/10398157/138093f6-6eb4-11e5-9f7f-d378550438db.gif)

**Comment by planemad** [comment]
**Update**
- Today we started on [Kyushu Island](http://tasks.openstreetmap.in/project/57) of around 22,000km2
- Estimated completion: Pass 1: 9 days, Pass 2: 4.5 days

**Comment by mapconcierge** [comment]
＞ @planemad @lxbarth @maning @Rub21 

I've translated @PlaneMad's post on diary of OSM.org in Japanese.
http://www.openstreetmap.org/user/MAPconcierge/diary/36106

![2015-10-14 14 09 15](https://cloud.githubusercontent.com/assets/416977/10475351/765ccb1c-727d-11e5-8bac-d3ed949cffc5.png)

**Comment by pratikyadav** [comment]
For selecting continous way use `Shift + W`. It grabs the continous active way upto the next intersection.

Fliter out `bridges` ,`tunnels` and minor roads.

![japan11](https://cloud.githubusercontent.com/assets/12744420/10481712/2c356a54-7290-11e5-8081-0aa97ba65d77.gif)

Just keep an eye on `tags` while merging.

**Comment by lxbarth** [comment]
> I've translated @PlaneMad's post on diary of OSM.org in Japanese.
> http://www.openstreetmap.org/user/MAPconcierge/diary/36106

*So amazing.* Thank you @mapconcierge !

**Comment by mapconcierge** [comment]
Your too :-)  ＞ @lxbarth 
Thanks for great challenge for Asian mappers.

**Comment by planemad** [comment]
@mapbox/team-data to speed up segment merging, JOSM can be configured to discard the `yh:WIDTH` tag automatically when editing. To do this search for the `tags.discardable` property in [advanced JOSM settings](https://gist.github.com/planemad/517401649204c474c906) and add `yh:WIDTH` to the list.
<img width="624" alt="screenshot 2015-10-15 14 54 35" src="https://cloud.githubusercontent.com/assets/126868/10509789/ca267638-734c-11e5-9534-f4ccbd7b7fe5.png">

JOSM and iD will be updated in the next release to do this automatically for new users.

**Comment by planemad** [comment]
**Update**
<img width="395" alt="screenshot 2015-10-19 18 48 37" src="https://cloud.githubusercontent.com/assets/126868/10578802/0b2814d2-7692-11e5-80c9-46e0086a8246.png">
- Pass 1 of Kyushu complete: 19 members, 5 days
- Pass 2 estimate: 4.5 days (Thursday)

**Comment by pratikyadav** [comment]
## Update

Yesterday we (@pratikyadav @ramyaragupathy @maning @planemad ) had a chat with @mapconcierge and discussed briefly about the current status of this project, as well as the future course of actions. Here are notes on the conversation-


- Japan OSM community appreciate Mapbox efforts and are willing to contribute but, language barrier to effectively communicate is the problem.
- We need to improve the tasking instructions (use simple language) and translate to Japanese.
- Taichi  to prepare the next mega task for Tokyo in teachosm in coordination with pratik.
- Mapbox team will use teachosm TM in the next tasks.
- Most of the new users in Japan use iD
- Japan community is planning a new import within Osaka prefecture, we need to investigate how this will affect our fixes. http://wiki.openstreetmap.org/wiki/Tondabayashi/Road_Building_import
- Try to contact more core OSm members in Japan and discuss about this project.
- Japanese drone community is growing, encourage them to upload imagery to openaerialmap.org so we can use them for area without GSI coverage.
- Taichi in talks with GSI to extend the coverage to other areas.

**Comment by pratikyadav** [comment]
## Tokyo Mega Task

As per the previous discussion, we'll create the next mega task for Japan on [TeachOSM](http://tasks.teachosm.org/).
To make it simpler for the new users and community, we will be doing only pass 1 on this task.

#### Next Action-

- [ ] @mapconcierge to create Tokyo task using these [instruction](https://gist.github.com/pratikyadav/fcbcc24e3e13f866b8a4)
- [ ] @mapbox/team-data  to join the community in this task.
- [ ] OSM dairy entry in Japanese and English about this task (may be @mapconcierge and @maning )

**Comment by Andygol** [comment]
### 九州 Kyushu task - 57
Pass 1 and 2 done - http://tasks.openstreetmap.in/project/57#stats
![image](https://cloud.githubusercontent.com/assets/369696/10637487/9eb7df20-780e-11e5-9c62-da422f904bb0.png)

**Comment by mapconcierge** [comment]
Japanese Students are readying mega Tokyo Task by JOSM...

![12022351_1136401849722118_382764361824730268_o](https://cloud.githubusercontent.com/assets/416977/10662872/3eb075d0-78f2-11e5-9c9a-2d6de97176b0.jpg)

I will make a task-project on the TeachOSM's Tasking Manager tonight. :-)

**Comment by mapconcierge** [comment]
I've made a Tokyo Mega Task area as GeoJSON!
https://gist.github.com/mapconcierge/17c5c01d384191eeccac
![2015-10-23 12 24 13](https://cloud.githubusercontent.com/assets/416977/10683824/26bed964-7981-11e5-83e2-c641c54d5868.png)

Next step: making a Task-Project on TeachOSM Tasking Maneger.

**Comment by mapconcierge** [comment]
I've made Tokyo Mega Task on the TeachOSM Taking Manager.
http://tasks.teachosm.org/project/91

![2015-10-23 12 49 25](https://cloud.githubusercontent.com/assets/416977/10684117/bdcc9adc-7984-11e5-8204-b4128085ab08.png)

**Comment by maning** [comment]
### Filtering data in JOSM and iD

- JOSM users: Use inverse filter `highway=motorway|trunk|primary|secondary|*link|motorway_link`

![josm_filter](https://cloud.githubusercontent.com/assets/353700/10685384/066bd5f4-7977-11e5-9c2a-d072dab57128.gif)

- iD users: Use the `Map data > Filters`

![id_filter](https://cloud.githubusercontent.com/assets/353700/10685385/06a280fe-7977-11e5-88f1-95a0b289f0ef.gif)

**Comment by ramyaragupathy** [comment]
#### Random admin tags on major roads

![admintags_small](https://cloud.githubusercontent.com/assets/12103383/10685578/32ded2d8-7979-11e5-87ab-8c997a3655c6.gif)

Though they are part of some relation, these boundaries are not continuous.  How do we deal with these random boundary tags?

cc @maning @mapconcierge

**Comment by maning** [comment]
@mapconcierge Great!  

I modified the EN instructions since we agreed to focus on re-aligning major roads for now see [this](https://github.com/mapbox/mapping/issues/120#issuecomment-149796885) and [this](https://github.com/mapbox/mapping/issues/120#issuecomment-149798448).

I also created GIFs on how to filter major roads in iD and JOSM.  Ideally it should be in the TM project Instructions tab, it used to work before.  For now I just linked it in this ticket.

Please translate the new instructions in JP.  Our team will start working on this today. :rocket:

**Comment by maning** [comment]
> Though they are part of some relation, these boundaries are not continuous. How do we deal with these random boundary tags?

Do not merge or delete. What do you think @mapconcierge?

**Comment by maning** [comment]
Documenting a few issues now that we are using the [teachosm TM instance for Tokyo](http://tasks.teachosm.org/project/91):
* Changeset comment and source tag were not assigned automatically in JOSM. It works on iD.  **Make sure you copy the recommended changeset comment from the Instructions.**
* For very dense areas, JOSM complain that the data is too big even when working on the smallest possible tile. 
* However, for central Tokyo, the data looks very well aligned.  I suggest @mapbox/team-data focus on these areas so that other community mappers can work on the manageable chunk of data especially if they use iD. 
* @Andygol will translate instruction into Russian to involve OSM-RU (UA,BY and other exUSSR) community.
* @maning and @mapconcierge will prepare a diary to announce this task once translation (JP and RU) is done.

**Comment by pratikyadav** [comment]
https://www.openstreetmap.org/user/pratikyadav/diary/36182

Dairy entry up in English for Japan mega task.
@mapconcierge would be great if you or someone from Japan OSM community can translate this in Japanese.

**Comment by pratikyadav** [comment]
![screen shot 2015-10-27 at 2 49 04 pm](https://cloud.githubusercontent.com/assets/12744420/10757064/413bf88a-7cce-11e5-9e65-3eb975818835.png)

**Comment by mapconcierge** [comment]
Thanks @pratikyadav 

--
I've done too!! :-)
https://www.openstreetmap.org/user/MAPconcierge/diary/36201

Happy Mapping!

![2015-10-29 17 56 02](https://cloud.githubusercontent.com/assets/416977/10814056/822720d8-7e66-11e5-9076-aa20caf79b2f.png)

**Comment by pratikyadav** [comment]
@mapconcierge thank you :)

**Comment by mapconcierge** [comment]
＞ @pratikyadav 

I've sent a mail to Talk-ja ML. :-)

**Comment by mapconcierge** [comment]
I've fixed a wrong cross road in Sagamihara city, Kanagawa Pref.
http://www.openstreetmap.org/edit?editor=id&#map=20/35.59537/139.34157 

![crosspoint](https://cloud.githubusercontent.com/assets/416977/10814589/39c782de-7e6a-11e5-9194-3708938825f6.png)

**Comment by taroken** [comment]
I've fixed this street which had't done correctly.
![mistake](https://cloud.githubusercontent.com/assets/13146539/10814709/e1ec28d4-7e6a-11e5-8f9f-2424356d6caa.png)
![mistake2](https://cloud.githubusercontent.com/assets/13146539/10814710/e1f0cc04-7e6a-11e5-802e-1601ce8f7e12.png)

**Comment by kuma03** [comment]
I've found a area which has not checked yet and added
![default](https://cloud.githubusercontent.com/assets/13146562/10814763/385ed090-7e6b-11e5-8d30-ff1c3bf4e3a5.png)

**Comment by AyameO** [comment]
This is one of cross road in Sagamihara city, Kanagawa Pref. And I've fixed this wrong road.
![tasks](https://cloud.githubusercontent.com/assets/13360878/10814771/46f5dd56-7e6b-11e5-9c61-68c09ded2310.png)
![tasks](https://cloud.githubusercontent.com/assets/13360878/10814804/816beaca-7e6b-11e5-83df-bcc11610c835.png)

**Comment by Misa0716** [comment]
I've fixed a wrong cross road in Sagamihara city, Kanagawa Pre
![error2](https://cloud.githubusercontent.com/assets/13146549/10814863/d4af2698-7e6b-11e5-9ff6-6fda6c1fb5e3.png)
f.

**Comment by pratikyadav** [comment]
@taroken @kuma03 @AyameO @Misa0716 
This is great. Thank you so much for joining in. :tada:

**Comment by yuuuuka** [comment]
I found and fixed this road which displaced from correct road in Sagamihara city , Kanagawa 
Pref.
![wrong road](https://cloud.githubusercontent.com/assets/13146558/10815871/228bd1e4-7e72-11e5-9967-5c1d7de089ec.png)

**Comment by mirin0530** [comment]
I've fixed this road,which is out of center in Sagamihara, Kanagawa prefecture.

![strange_way](https://cloud.githubusercontent.com/assets/13146610/10837647/35a0dbfc-7eff-11e5-8081-e3dad4b72431.png)

**Comment by pratikyadav** [comment]
@yuuuuka @mirin0530 Thank you :+1:

**Comment by pratikyadav** [comment]
### Progress update 10/30

Currently working on- [Tokyo Mega task](http://tasks.teachosm.org/project/91) on TeachOSM

Start date- 24th October
Estimated time of completion - 1 more week.

![screen shot 2015-10-30 at 2 26 19 pm](https://cloud.githubusercontent.com/assets/12744420/10841771/3c219878-7f12-11e5-9ad2-b26db671cb83.png)

:trophy:  Dairy entry in English and Japanese for Tokyo Mega task
:trophy:  Large Japan community  involvement 

Next action-
- [ ] @pratikyadav to make task boundary for Nagoya and share it with @mapconcierge for next mega task

**Comment by calfarome** [comment]
Team progress in  Japan.
![japant](https://cloud.githubusercontent.com/assets/10413117/10871980/379d392e-80c5-11e5-98ae-cff31b61df42.png)

**Comment by pratikyadav** [comment]
## Nagoya Mega task

As of today, Tokyo mega task is 68% complete. The next task in this series will be **Nagaya** and nearby area upto Tokyo.
 
![screen shot 2015-11-02 at 12 32 34 pm](https://cloud.githubusercontent.com/assets/12744420/10875934/fb329d50-815d-11e5-8883-9b352a323e90.png)

Next Action-
- [ ] @mapconcierge to make Nagoya task on TeachOSM using this geojson of extend of GSI imagery coverage. - > https://www.dropbox.com/s/tba6vj5nweicdd4/Nagoya.geojson?dl=0 (The Task size to be XS)
- [ ] @pratikyadav and @maning  to add instruction in English.
- [ ] @Andygol  to add instruction in Russian.
- [ ] @abel801  to add instruction in Spanish.

**Comment by maning** [comment]
@pratikyadav, can we run an initial analysis to check which parts of Nagoya mega-task needs focused work? I'm thinking we can use TM's priority areas option.  For example, a criteria maybe either:
1. densely populated areas, or;
2. untouched ways after the import.

cc/ @planemad @mapconcierge

**Comment by pratikyadav** [comment]
> densely populated areas

The [tweet map](http://rub21.github.io/top-tiles/app/) by @Rub21  could be very helpful in identification of priority areas.

![screen shot 2015-11-04 at 4 50 58 pm](https://cloud.githubusercontent.com/assets/12744420/10936707/42d0f168-8314-11e5-8bcc-a5c24de86140.png)

**Comment by mapconcierge** [comment]
Great works!! :-)

＞ @pratikyadav 
Okay, I will make Nagoya Mega Task soon!

**Comment by mapconcierge** [comment]
I've made a Nagoya Mega Task!!
http://tasks.teachosm.org/project/104#

![2015-11-05 1 25 30](https://cloud.githubusercontent.com/assets/416977/10944368/328ec3d2-835c-11e5-928b-513a7cb73622.png)

**Comment by NatsukiTashiro** [comment]
I found and fixed this road in Nagoya city, Aichi pref.
![image](https://cloud.githubusercontent.com/assets/13250654/10960755/da51c50c-83c9-11e5-90e0-a60a8b7f815c.png)

**Comment by pratikyadav** [comment]
@mapconcierge Great work :tada: 

@ramyaragupathy  can you write a OSM diary entry about Nagoya task and stats of Tokyo task in English?

**Comment by pratikyadav** [comment]
## Progress Update

Completed Tokyo Mega task- http://tasks.teachosm.org/project/91#

![screen shot 2015-11-05 at 2 05 46 pm](https://cloud.githubusercontent.com/assets/12744420/10963449/857be95c-83c6-11e5-97df-71cad4687dea.png)


![screen shot 2015-11-05 at 2 04 35 pm](https://cloud.githubusercontent.com/assets/12744420/10963454/8e69c8fe-83c6-11e5-948e-1bbaf54317bf.png)

cc @lxbarth @maning @planemad @mapconcierge

**Comment by planemad** [comment]
This is HUGE, amazing work everyone!

**Comment by ramyaragupathy** [comment]
>@ramyaragupathy  can you write a OSM diary entry about Nagoya task and stats of Tokyo task in English?

There is a diary entry up for Nagoya task now. http://www.openstreetmap.org/user/ramyaragupathy/diary/36249. 
Next action on @mapconcierge to get a Japanese version out!

**Comment by shvrm** [comment]
MASSIVE team effort! Great work everyone :japan: !

**Comment by maning** [comment]
> - Japan community is planning a new import within Osaka prefecture, we need to investigate how this will affect the current improvement activities. http://wiki.openstreetmap.org/wiki/Tondabayashi/Road_Building_import

In preparation for the new Osaka task, we will exclude the main Osaka City which [we already did](http://tasks.openstreetmap.in/project/54).  We will include Tondabayashi, since the planned import will not incldue major roads that we are doing the realignment.

@mapconcierge

**Comment by Andygol** [comment]
In case you faced with warning that JOSM can't download area (a lot of data) - use next overpass query for downloading data

```
(
  way[highway];>;
  rel[highway];>;
  way[railway];>;
);
(._;rel(bn););
(._;rel(bw););
(._;rel(br););
out meta qt;
```

**Comment by planemad** [comment]
Have filed an issue in iD to prompt alternate imagery based on map location: https://github.com/openstreetmap/iD/issues/2826

This is an important issue to fix if we want new users to trace and realign from the correct source as we cannot expect them to know the availability of GSI and the tileurl.

**Comment by lxbarth** [comment]
> Completed Tokyo Mega task

HUGE! @pratikyadav - we should diary-post this in Japanese and English. Maybe we can get @mapconcierge's help?

**Comment by shvrm** [comment]
:boom: @pratikyadav!

**Comment by pratikyadav** [comment]
https://www.openstreetmap.org/user/pratikyadav/diary/36280
OSM diary entry up.

cc @lxbarth @maning @planemad @mapconcierge

**Comment by abel801** [comment]
@pratikyadav  I don't know where can I translate the instructions, in a gist or do I have to have access to the task?

**Comment by pratikyadav** [comment]
@abel801 you can make a gist, and i'll update them on the task

**Comment by abel801** [comment]
@pratikyadav the translation  to Spanish is  [ Here](https://gist.github.com/abel801/fc3de07e1c30ad9f171b)

**Comment by mapconcierge** [comment]
I've made a Osaka Mega Task now!! Boom! :-)
http://tasks.teachosm.org/project/105

![2015-11-11 17 25 21](https://cloud.githubusercontent.com/assets/416977/11086965/9c2fa36e-8899-11e5-9404-9b6412ba3ea9.png)

**Comment by maning** [comment]
Nagoya DONE! :rocket: 

Next task is **Mega Osaka: http://tasks.teachosm.org/project/105**
Project is available in EN, ES, JP and RU languages.  Again, a some reminders when you edit:
* Before aligning, use the [combine-highways script](https://github.com/mapbox/mapping/tree/master/JOSM/scripts/combine-highways) in JOSM to merge highways.  Then, upload.
* Changeset comment and source tag were not assigned automatically in JOSM. It works on iD.  **Make sure you copy the recommended changeset and source comment from the Instructions.**

**Comment by lxbarth** [comment]
> Nagoya DONE! :rocket: 

Incredible.

**Comment by pratikyadav** [comment]
We are almost done with Osaka. (21% left)

We are extending the list of mega tasks to one more by including Hiroshima and thus will be covered with most of the east coast coverage of GSI imagery.

link of geojson for Hiroshima task -> https://www.dropbox.com/s/y4l84qw2bjtrlcm/Hiroshima.geojson?dl=0 

Next action

- [ ] @mapconcierge to make task in teach OSM
- [ ] @pratikyadav to add instructions in ru, es and en
- [ ] @nikhilprabhakar  to post diary entry about the Japan progress

**Comment by mapconcierge** [comment]
I found this interest area in Nara city in OSAKA mega task.
But I guess, Under construction area. We shouldn't modify.
http://www.openstreetmap.org/edit?editor=id&#map=18/34.68176/135.80939
![2015-11-19 9 53 49](https://cloud.githubusercontent.com/assets/416977/11259318/030a5ab0-8ea4-11e5-8611-35780e553624.png)

**Comment by mapconcierge** [comment]
＞ @pratikyadav 
Okay, I will make a Hiroshima Task soon.
Wait a moment... :-)

**Comment by hackablemeister** [comment]
This area is off to the left side over a wide range in Matsuzaka city.
http://www.openstreetmap.org/edit?editor=id&#map=17/34.54174/136.55632
![Uploading Matsuzaka_city.png…]()

**Comment by wdolphin** [comment]
I found someone edited wrongly in Nara city, Osaka mega task.  So I fixed.

https://cloud.githubusercontent.com/assets/15658355/11259536/b4cd85f0-8ea5-11e5-8057-783cd724f963.JPG

https://cloud.githubusercontent.com/assets/15658355/11259538/b90e8272-8ea5-11e5-874c-87210a586e43.JPG

**Comment by hackablemeister** [comment]
This area is off to the left side over a wide range in Matsuzaka city.
http://www.openstreetmap.org/edit?editor=id&#map=17/34.54174/136.55632
<img width="884" alt="matsuzaka_city" src="https://cloud.githubusercontent.com/assets/13362658/11259644/93047e64-8ea6-11e5-88b8-fb4885d5ebca.png">

**Comment by hageshi1** [comment]
I have no idea , how to fix? :-)
https://gyazo.com/09b3309a9a7daec272403490b837d08b
http://www.openstreetmap.org/edit?editor=id&#map=19/34.70021/135.81981

**Comment by mapconcierge** [comment]
＞ @abel801  Muito Bom!!

＞ @planemad 
Hiroshima Task!! http://tasks.teachosm.org/project/111
![2015-11-20 7 38 06](https://cloud.githubusercontent.com/assets/416977/11286747/2aec47c6-8f5a-11e5-959d-1410ad22f103.png)

**Comment by pratikyadav** [comment]
@mapconcierge  Great :tada:  
@hageshi1 Semms like a bridge connecting to a nearby parking area. Will be nice to add a `layer=1` tag to this road.

**Comment by mapconcierge** [comment]
Thanks!! @pratikyadav 
I've confirmed!! :-)

![2015-12-26 11 05 09](https://cloud.githubusercontent.com/assets/416977/12004951/99d0dcfa-abc6-11e5-950d-476db316678e.png)

**Comment by mapconcierge** [comment]
* https://github.com/osmlab/editor-imagery-index/issues/111
* https://github.com/osmlab/editor-imagery-index/pull/117

![2015-12-26 11 57 58](https://cloud.githubusercontent.com/assets/416977/12004966/081dd7ac-abc8-11e5-865f-25179a6f19e0.png)

**Comment by planemad** [comment]
@mapconcierge many thanks for all the help from the Japanese community in the task. We will continue to interact on our diary posts and the mailing lists.

No immediate actions here. Amazing job everyone!
:rocket: :rocket: :rocket: 

cc @mapbox/team-data

**Comment by talllguy** [comment]
@mapconcierge I stumbled across this highway system overlapping water in #103. Since it is related to Japan, I thought you might want to check it out. Should that street network be on top of that water feature?

[map link](https://www.openstreetmap.org/way/206721948#map=15/38.5622/141.4538&layers=C)

![2016-03-01 17_33_04-java openstreetmap editor](https://cloud.githubusercontent.com/assets/1827881/13444356/b80905b2-dfd3-11e5-9137-109aa89c31d0.png)

**Comment by jeffrey-friedl** [comment]
Is this project still ongoing?  In many cases it's possible to use road-edge data from the Japanese government to guide extremely accurate updates that are not subject to image misalignment or elevation-change skew. I've been doing this around Kyoto a lot, but it'd be nice if more folks could do it.

**Comment by miurahr** [comment]
> @mapconcierge I stumbled across this highway system overlapping water in #103. Since it is related to Japan, I thought you might want to check it out. Should that street network be on top of that water feature?

There were actually a many roads in this area. When the  earthquake / tsunami in 2011 caught here, many of this area, Kamatani and Nagato region, fully submerged.

We should mark these roads as non-usable or lost.

**Comment by pratikyadav** [comment]
> Is this project still ongoing?

@jeffrey-friedl No, this is not an on-going project. But I would love to learn more about road-edge data from the Japanese government. Can you provide some relevant link?

**Comment by jeffrey-friedl** [comment]
You may need to create a (free) account to access this link, but the data is downloadable at:

http://fgd.gsi.go.jp/download/kibansel.php

It's the "道路縁" field.   The data is difficult to work with, but in the end you can turn in into a GPX/KML file that ends up drawing the edges of roads. Using that as a guide, a human can infer road paths.

Here's an example that you can load into an openstreetmap editor or Google Earth or other visualizer:  http://regex.info/tmp/road-edge-sample.gpx

The data is usually very high quality, but occasionally it is out of date due to recent construction. Also, in some areas the data seems more like an old planned road, rather than actual construction. But mostly it's spot on correct.

I came across your project because it updated a road I had used this technique with. The project update broke map correctness because the GSI image from which your update was based was old... the road had since been move.  (I'm still trying to get that update, changlist #35265050, reverted.)
